src/v2/System/locales/en/translation.json

```diff
@@ -0,0 +1,22 @@
+{
+  "navbar":{
+    "show":"Show",
+    "shows":"Shows",
+    "buy":"Buy",
+    "artists":"Artists",
+    "artworks":"Artworks",
+    "galleries":"Galleries",
+    "auctions":"Auctions",
+    "fairs":"Fairs",
+    "museums":"Museums",
+    "viewingRooms":"Viewing Rooms",
+    "nfts":"NFTs",
+    "signup":"Sign up",
+    "login":"Log In",
```

Comment:
Two things:

- Wondering about difference in casing between these two phrases. Intentional? (Looks they were both title-case before, but now "Sign up" is sentence-case.)

- Noting [the fallout](https://artsy.slack.com/archives/C9SATFLUU/p1658171312694339) in Integrity specs that we're seeing in CI now, related to this change

Reply:
Yep, should be title case

Reply:
Fixed over here: https://github.com/artsy/force/pull/10550